README.md

```diff
@@ -17,9 +29,6 @@ The goal of the RAPIDS Memory Manager (RMM) is to provide:
 For information on the interface RMM provides and how to use RMM in your C++ code, see
 [below](#using-rmm-in-c).
 
-**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/rmm/blob/main/README.md)
-ensure you are on the `main` branch.
```

Comment:
Is this no longer true?

Reply:
This line was moved, to match the order of the cuDF README. Users wanting the stable README (from the last release) should look at the `main` branch instead of the default branch that shows on GitHub (currently `branch-22.10`).